problems.rb

```diff
@@ -0,0 +1,85 @@
+# Problem #1
+# Write a method named print_array that accepts an array of integers as a parameter and that prints the contents of the array as a bracketed, comma-separated list.
+# For example, if a variable called list stores the following values: list = [3, 19, 27, 4, 98, 304, -9, 72]
+# Then the call of print(list) should produce the following output: [3, 19, 27, 4, 98, 304, -9, 72]
+# Your method should produce a single line of output (may wrap with long lists).
+#
+def print_array(int_array)
+  list = []
+  int_array.each do |int|
+    list << int
+  end
+  return list
+end
```

Comment:
This works, but isn't what I intended. I wanted you to manually create the string in the format provided. See Madeline's solution as an example.